packages/cli/Cargo.toml

```diff
@@ -7,7 +7,7 @@ description = "CLI for building fullstack web, desktop, and mobile apps with a s
 repository = "https://github.com/DioxusLabs/dioxus/"
 license = "MIT OR Apache-2.0"
 keywords = ["mobile", "gui", "cli", "dioxus", "wasm"]
-rust-version = "1.79.0"
+rust-version = "1.81.0"
```

Comment:
swc requires a newer version of rust to build. I'm not sure how much we care about the rust version the CLI requires since we are leaning into binstall everywhere, but if this an issue I could look for alternatives